GPU available: True, used: True
TPU available: False, using: 0 TPU cores
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name      | Type              | Params
------------------------------------------------
0 | criterion | BCEWithLogitsLoss | 0     
1 | cnn1      | ResNet            | 25.6 M
2 | fc1       | Sequential        | 4.3 M 
------------------------------------------------
29.8 M    Trainable params
0         Non-trainable params
29.8 M    Total params
119.248   Total estimated model params size (MB)
Traceback (most recent call last):
  File "./src/Main.py", line 50, in <module>
    
  File "./src/Main.py", line 39, in main
    progress_bar_refresh_rate=config['progress_bar_refresh_rate'])
  File "/home/hpc/iwi5/iwi5012h/dev/jigsaw-puzzle-solver/env/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py", line 458, in fit
    self._run(model)
  File "/home/hpc/iwi5/iwi5012h/dev/jigsaw-puzzle-solver/env/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py", line 756, in _run
    self.dispatch()
  File "/home/hpc/iwi5/iwi5012h/dev/jigsaw-puzzle-solver/env/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py", line 797, in dispatch
    self.accelerator.start_training(self)
  File "/home/hpc/iwi5/iwi5012h/dev/jigsaw-puzzle-solver/env/lib/python3.7/site-packages/pytorch_lightning/accelerators/accelerator.py", line 96, in start_training
    self.training_type_plugin.start_training(trainer)
  File "/home/hpc/iwi5/iwi5012h/dev/jigsaw-puzzle-solver/env/lib/python3.7/site-packages/pytorch_lightning/plugins/training_type/training_type_plugin.py", line 144, in start_training
    self._results = trainer.run_stage()
  File "/home/hpc/iwi5/iwi5012h/dev/jigsaw-puzzle-solver/env/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py", line 807, in run_stage
    return self.run_train()
  File "/home/hpc/iwi5/iwi5012h/dev/jigsaw-puzzle-solver/env/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py", line 842, in run_train
    self.run_sanity_check(self.lightning_module)
  File "/home/hpc/iwi5/iwi5012h/dev/jigsaw-puzzle-solver/env/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py", line 1107, in run_sanity_check
    self.run_evaluation()
  File "/home/hpc/iwi5/iwi5012h/dev/jigsaw-puzzle-solver/env/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py", line 934, in run_evaluation
    self.evaluation_loop.on_evaluation_start()
  File "/home/hpc/iwi5/iwi5012h/dev/jigsaw-puzzle-solver/env/lib/python3.7/site-packages/pytorch_lightning/trainer/evaluation_loop.py", line 82, in on_evaluation_start
    self.trainer.call_hook('on_validation_start', *args, **kwargs)
  File "/home/hpc/iwi5/iwi5012h/dev/jigsaw-puzzle-solver/env/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py", line 1223, in call_hook
    trainer_hook(*args, **kwargs)
  File "/home/hpc/iwi5/iwi5012h/dev/jigsaw-puzzle-solver/env/lib/python3.7/site-packages/pytorch_lightning/trainer/callback_hook.py", line 222, in on_validation_start
    callback.on_validation_start(self, self.lightning_module)
  File "/home/hpc/iwi5/iwi5012h/dev/jigsaw-puzzle-solver/env/lib/python3.7/site-packages/pytorch_lightning/callbacks/progress.py", line 422, in on_validation_start
    reset(self.val_progress_bar, sum(trainer.num_sanity_val_batches))
  File "/home/hpc/iwi5/iwi5012h/dev/jigsaw-puzzle-solver/env/lib/python3.7/site-packages/pytorch_lightning/callbacks/progress.py", line 515, in reset
    bar.reset(total=convert_inf(total))
AttributeError: 'tqdm' object has no attribute 'reset'
